gwt-regexp/pom.xml

```diff
@@ -65,39 +67,26 @@
             <version>${elemental2.version}</version>
             <classifier>sources</classifier>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
 
     <build>
-        <resources>
-            <resource>
-                <directory>src/main/java</directory>
-                <includes>
-                    <include>**/regexp/**</include>
-                </includes>
-            </resource>
-            <resource>
-                <directory>src/main/resources</directory>
-                <includes>
-                    <include>**/regexp/**</include>
-                    <include>**/*.gwt.xml</include>
-                    <include>**/*.html</include>
-                    <include>**/public/**</include>
-                </includes>
-            </resource>
-        </resources>
-
         <plugins>
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <version>${maven.compiler.plugin}</version>
+                <groupId>net.ltgt.gwt.maven</groupId>
+                <artifactId>gwt-maven-plugin</artifactId>
+                <version>${maven.gwt.plugin}</version>
+                <extensions>true</extensions>
                 <configuration>
-                    <encoding>${project.build.sourceEncoding}</encoding>
-                    <source>${maven.compiler.source}</source>
-                    <target>${maven.compiler.target}</target>
+                    <moduleName>org.gwtproject.regexp.RegExp</moduleName>
+                    <moduleShortName>gwtregexp</moduleShortName>
```

Comment:
unneeded, since we never compile to js output

Reply:
agreed.